Config.uk

```diff
@@ -236,29 +236,78 @@ config LWIP_DNS_TABLE_SIZE
 	default 32
 endif
 
-config LWIP_SOCKET
-	bool "Socket API"
-	select LIBVFSCORE
+menu "Socket"
```

Comment:
Can you use menuconfig instead? This basically combines the menu with your `LWIP_SOCKET` option.